digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/ObstacleService.scala

```diff
@@ -54,9 +65,9 @@ class ObstacleService(val roadLinkService: RoadLinkService) extends PointAssetOp
     createPersistedAsset(asset, adjustment)
   }
 
-  private def adjustmentOperation(persistedAsset: PersistedAsset, adjustment: AssetAdjustment): Long = {
+  private def adjustmentOperation(persistedAsset: PersistedAsset, adjustment: AssetAdjustment, roadLink: RoadLink): Long = {
     val updated = IncomingObstacle(adjustment.lon, adjustment.lat, adjustment.linkId, persistedAsset.obstacleType)
-    OracleObstacleDao.update(adjustment.assetId, updated, adjustment.mValue, "vvh_generated", persistedAsset.municipalityCode, Some(adjustment.vvhTimeStamp), persistedAsset.linkSource)
+    updateWithoutTransaction(adjustment.assetId, updated, roadLink.geometry, persistedAsset.municipalityCode, "vvh_generated", persistedAsset.linkSource)
   }
```

Comment:
Maybe "Some(adjustment.vvhTimeStamp)" shouldn't be excluded  from update (this is related with changeInfo.vvhTimeStamp)